src/goInstallTools.ts

```diff
@@ -22,6 +22,7 @@ let tools: { [key: string]: string } = {
 	'golint': 'github.com/golang/lint/golint',
 	'go-outline': 'github.com/lukehoban/go-outline',
 	'go-symbols': 'github.com/newhook/go-symbols',
+	'goimports': 'golang.org/x/tools/cmd/goimports',
```

Comment:
Why is this needed?  Goreturns is the default tool, which should be a superset of `goimports`.  Is there an issue tracking a scenario for which this is needed (and for which we should have all vscode-go users download this additional binary?)